user/pounce/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "pounce"
+pkgver = "3.1"
+pkgrel = 0
+build_style = "configure"
```

Comment:
gnu_configure

Reply:
Done.

I'm curious, why use `gnu_configure` over `configure` in this case? I was mostly using the [catgirl](https://github.com/chimera-linux/cports/blob/master/main/catgirl/template.py) package as a reference. Does that package need an update? When would one want to use `configure` over `gnu_configure`?

Reply:
I guess if `gnu_configure` works, it's preferred over the bundled `configure`
as the automake/autoconf files are infinitely more readable than the
`configure` file that's produced by them.

Reply:
@subnut The `configure` script in catgirl is also manually written and is almost identical to the one in pounce.

Reply:
oh, and using `gnu_configure` also produces this output during the `configure` build phase, I assume these warnings can be ignored?

```
warning: unsupported option --sysconfdir=/etc
warning: unsupported option --sbindir=/usr/bin
warning: unsupported option --libdir=/usr/lib
warning: unsupported option --infodir=/usr/share/info
warning: unsupported option --localstatedir=/var
warning: unsupported option --build=x86_64-chimera-linux-musl
warning: unsupported option --host=x86_64-chimera-linux-musl
```